docs/source/models.rst

```diff
@@ -87,6 +90,61 @@ signature
 input_example
   reference to an artifact with :ref:`input example <input-example>`.
 
+databricks_runtime
+    Databricks runtime version and type, if the model was trained in a Databricks notebook or job.
+
+
+
+
+Additional Logged Files
+^^^^^^^^^^^^^^^^^^^^^^^
+For environment recreation, we automatically log ``conda.yaml`` and ``requirements.txt`` files whenever a model is logged. These files can then be used to reinstall dependencies using either ``conda`` or ``pip``.
+
+conda.yaml
+    When saving a model, MLflow provides the option to pass in a conda environment parameter that can contain dependencies used by the model. If no conda environment is provided, a default environment is created based on the flavor of the model. This conda environment is then saved in ``conda.yaml``.
+requirements.txt
+    The requirements file is created from the `pip portion <https://www.anaconda.com/blog/using-pip-in-a-conda-environment>`_ of the ``conda.yaml`` environment specification. Additional pip dependencies can be added to ``requirements.txt`` by including them as a pip dependency in a conda environment and logging the model with the environment. 
+
+The following shows an example of saving a model with a specified conda environment and the corresponding content of the generated ``conda.yaml`` and ``requirements.txt`` files.
```

Comment:
maybe "specified" -> "manually specified" to make it more clear that you don't need to do this? (we do say that above but we can't expect someone to read that once, probably briefly, and have it register :-))

Reply:
+1 to this yeah